docs/output.md

```diff
@@ -2,42 +2,79 @@
 
 ## Introduction
 
-This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+This document describes the output produced by the pipeline.
 
 The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
-
 ## Pipeline overview
 
 The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
 
-- [FastQC](#fastqc) - Raw read QC
+- [Alignments](#alignments) - Alignment of the _query_ genomes to the _target_ genome
+- [Dot plots](#dot-plots) - Visualisation of the alignment of the _query_ genomes to the _target_ genome
+- [`N` regions](#n-regions) - Coordinate of the `N` regions on the _query_ and _target_ genomes
 - [MultiQC](#multiqc) - Aggregate report describing results and QC from the whole pipeline
 - [Pipeline information](#pipeline-information) - Report metrics generated during the workflow execution
 
-### FastQC
+Each _query_ genome, is aligned to the _target_ genome, and each alignment is visualised with dot plots. The output file names are constructed by concatenating the _target_ and _query_ sample identifiers with a `___` separator (three underscores), to faciliate re-extraction of the IDs from file names.
+
+### Assembly statistics
```

Comment:
```suggestion
## Assembly statistics
```
If you decrease the heading level of this and the following headings to h2, the will show up in the ToC on the side. 🙂